aws/amazon-chime-sdk-js
# Add messaging session

**Description of changes:**
Adding new APIs to create a web socket connection to receive chime chat message. 
**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes? 
- Unit test
- Manually test using the new demo app
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? 
Yes, the changes can be tested using the new demo app found under demos/browser/app/messagingSession.
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? Yes
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? Yes


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
